src/modules/visTypes.js

```diff
@@ -49,6 +52,9 @@ export const visTypeDisplayNames = {
     [VIS_TYPE_YEAR_OVER_YEAR_COLUMN]: i18n.t('Year over year (column)'),
     [VIS_TYPE_SINGLE_VALUE]: i18n.t('Single value'),
     [VIS_TYPE_SCATTER]: i18n.t('Scatter'),
+    [VIS_TYPE_LINE_LIST]: i18n.t('Line list'),
```

Comment:
This will break DV because of https://github.com/dhis2/data-visualizer-app/blob/eb4b868284491979c1b09bea56c5ea8eb697ca7f/packages/app/src/components/VisualizationTypeSelector/VisualizationTypeSelector.js#L57. Should be added to the breaking change notes and would be good to come up with a solution for this in DV before merging.

Reply:
Yep, the plan is to move this from Draft when both DV and ER PRs are ready.
I will check also Maps, but I think for that one there should not be any breaking change.